staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas_test.go

```diff
@@ -238,3 +238,200 @@ func testSchema() *schema.Structural {
 	}
 	return ts
 }
+
+func TestMaxLengths(t *testing.T) {
```

Comment:
How about having a single EstimateMaxLengths test function that covers both the cases where a maximum is defined on the schema and the cases where it is calculated (just label in the test cases which  is which)?

Reply:
Definitely makes more sense that way, fixed